include/forcing/DataProviderSelectors.hpp

```diff
@@ -134,7 +134,9 @@ class CSVDataSelector : public CatchmentAggrDataSelector
         CatchmentAggrDataSelector(std::string(), var, start, dur, units)
     {}
 
+    #if GCC_VERSION < 8 && !defined(__llvm__)
```

Comment:
FInally tried this with GCC6... I did not see the warnings and it compiled... so it looks like 8 is the right version to break this on (maybe 7 but 🤷 ).